optimum/exporters/onnx/config.py

```diff
@@ -75,10 +76,12 @@ class TextSeq2SeqOnnxConfig(OnnxSeq2SeqConfigWithPast):
 
     @property
     def inputs(self) -> Mapping[str, Mapping[int, str]]:
-        common_inputs = {
-            "input_ids": {0: "batch_size", 1: "encoder_sequence_length"},
-            "attention_mask": {0: "batch_size", 1: "encoder_sequence_length"},
-        }
+        common_inputs = OrderedDict(
```

Comment:
You do not need to convert the inputs to `OrderedDict`.  @michaelbenayoun uses the following function [config.ordered_inputs(model)](https://github.com/huggingface/optimum/blob/main/optimum/exporters/onnx/convert.py#L302) to convert the inputs into `OrderedDict` based on the input signature of the model.

Reply:
What's a good reason of having `ordered_inputs`? cc @michaelbenayoun  I noticed it failing silently here: https://github.com/huggingface/optimum/issues/569#issuecomment-1348168536

I added it because I would like to enforce the inputs to `add_past_key_values` to be an `OrderedDict`.

Reply:
I makes writing the `inputs` property easier because you dont have to think when to add which input to the dictionary, or what happens when you update it.

Reply:
all good